Description: fix segfaults due to wrong systemd service file
 The service file has the following line:
 User=transmission
 .
 It should be replaced with:
 User=debian-transmission
 .
 Bug-Debian: https://bugs.debian.org/718624
 Origin: other, https://bugs.debian.org/718624
 Forwarded: no
 Last-Update: 2014-10-16

---
 daemon/transmission-daemon.service | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

--- a/daemon/transmission-daemon.service
+++ b/daemon/transmission-daemon.service
@@ -3,7 +3,7 @@
 After=network.target
 
 [Service]
-User=transmission
+User=debian-transmission
 Type=notify
 ExecStart=/usr/bin/transmission-daemon -f --log-error
 ExecReload=/bin/kill -s HUP $MAINPID
